addons/website_event/views/event_templates_list.xml

```diff
@@ -149,9 +149,20 @@
     <!-- No events -->
     <t t-if="not event_ids">
         <div class="col-12">
-            <div class="h2 mb-3">No events found.</div>
-            <div class="alert alert-info text-center" groups="event.group_event_user">
-                <p class="m-0">Use the top button '<b>+ New</b>' to create an event.</p>
+            <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                <div class="mb-3 o_event_svg">
+                    <t t-call="website_event.event_svg"/>
+                </div>
+                <h2 class="mb-2">No events scheduled yet</h2>
+                <p>We couldn’t find any events scheduled at this moment.</p>
+            </div> 
+            <div class="o_not_editable d-flex flex-column align-items-center justify-content-between my-3" groups="event.group_event_user">
+                <div class="d-flex flex-column align-items-center justify-content-between">
+                    <a class="btn btn-secondary"> <!-- Add link to event modal creation -->
```

Comment:
`onClick="() => this.onClickNewContent(event)"` ?
Not sure, eventually check with the devs

Reply:
You have check this as "resolved", it's a mistake?